pages/guideforms.py

```diff
@@ -55,6 +55,17 @@ def validate(self, value):
   'Write it from a web developer\'s point of view.\n\n'
   'Follow the example link below for more guidance.')
 
+# This email address pattern is more restrictive than actually allowed, but it covers most reasonable uses.
+EMAIL_ADDRESS_PATTERN = '[ ]*([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9\-\.]+)\.([a-zA-Z]{2,5}){1,25}[ ]*'
+
+EMAIL_ADDRESSES_PATTERN = '^' + EMAIL_ADDRESS_PATTERN + '(,' + EMAIL_ADDRESS_PATTERN + ')*$'
```

Comment:
I don't think that you need the `^` and the `$` because those are implicit.
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern#constraint_validation

Reply:
I missed that. Thanks.